docs/registry/entities/telemetry.md

```diff
@@ -0,0 +1,77 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+# Telemetry
+
+## Telemetry Distro
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `telemetry.distro`
+
+**Description:** The distribution of telemetry SDK used to capture data recorded by the instrumentation libraries.
+
+**Other Attributes:**
+
+> :warning: This entity definition contains attributes without a role.
+> Stable Entities MUST NOT have attributes without a defined role.
```

Comment:
Is it documented that this means adding either of the following role property to the entity (eg. in `model/telemetry/entities.yaml`)?
```diff
+         role: identifying
```
or
```diff
+         role: descriptive
```

Reply:
That should be in the modelling guide: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/how-to-write-conventions/resource-and-entities.md